nova/scheduler/filters/shard_filter.py

```diff
@@ -24,6 +25,8 @@
 
 CONF = nova.conf.CONF
 
+HANA_USE_SHARDING = "use_individual_shard_tags_hana"
+
```

Comment:
This should move into the `ShardFilter` class if we do not use it outside this file (same as with `_ALL_SHARDS`).

Additionally, I think we have to add that tag to `ShardFilter._PROJECT_TAG_TAGS` for the `ProjectTagMixin` to consider it.